Astral Pathfinder/Models/Galaxy.hpp

```diff
@@ -15,6 +15,7 @@
 // MARK: Source Files
 #include "parameters.h"
 #include "GameObject.hpp"
+#include "Planet.hpp"
 
 
 class Galaxy {
```

Comment:
maybe rename to PlanetManager or something suitable (Galaxy might be fine), should have init functions for creating Homeworld and the rest of the galaxy's planets, also should be able to manage all the planets with an update function